### ISSUE TITLE
[Web] onStart event for Pan and Tap gestures not invoked

### ISSUE THREAD

**Issue by itsramiel** [software-mansion/react-native-gesture-handler#2057]
## Description

I am trying to run code when onStart event is invoked for Pan or Tap gesture. Both of them seem to not work independently of each other. 

### Platforms

- [ ] iOS
- [ ] Android
- [x] Web

### Screenshots
pan:
![image](https://user-images.githubusercontent.com/80689446/168982686-e5eae910-fc84-4e61-b333-8f5427a6712a.png)
tap:
![image](https://user-images.githubusercontent.com/80689446/168983835-7359ea92-05a4-4e23-85dc-cac716265a15.png)

## Steps To Reproduce

1. Pass a tap/pan gesture to GestureDetector and run some code in the onStart event's callback

### Expected behavior

I expect the onStart callback to be called

### Actual behavior

onStart callback is not being called

## Snack or minimal code example

pan:
repro repo: https://github.com/ramiel1999/gesture-handler-pan-tap/tree/pan-start-state-not-called-on-web
minimal code: 
```
export default function App() {
  const translateX = useSharedValue(0);
  const context = useSharedValue(0);

  const rStyle = useAnimatedStyle(() => {
    return { transform: [{ translateX: translateX.value }] };
  }, []);

  const panGesture = Gesture.Pan()
    .onStart(() => {
      console.log("onStart");
      context.value = translateX.value;
    })
    .onChange((e) => {
      console.log("onChange");
      translateX.value = e.translationX + context.value;
    })
    .onEnd(() => {
      console.log("onEnd");
      context.value = translateX.value;
    });

  return (
    <GestureHandlerRootView style={styles.container}>
      <GestureDetector gesture={panGesture}>
        <Animated.View style={[styles.box, rStyle]} />
      </GestureDetector>
    </GestureHandlerRootView>
```

tap:
repro repo: https://github.com/ramiel1999/gesture-handler-pan-tap/tree/tap-start-state-not-called-on-web
minimal code: 

```
export default function App() {
  const translateX = useSharedValue(0);

  const rStyle = useAnimatedStyle(() => {
    return { transform: [{ translateX: translateX.value }] };
  }, []);

  const tapGesture = Gesture.Tap()
    .onStart(() => {
      console.log("onStart");
      translateX.value = withTiming(100);
    })
    .onEnd(() => {
      console.log("onEnd");
      translateX.value = withTiming(-100);
    });

  return (
    <GestureHandlerRootView style={styles.container}>
      <GestureDetector gesture={tapGesture}>
        <Animated.View style={[styles.box, rStyle]} />
      </GestureDetector>
    </GestureHandlerRootView>
  );
}
```

## Package versions

- React: 17.0.2
- React Native: 0.68.2
- React Native Gesture Handler: 2.2.1
- React Native Reanimated: 2.8.0
- React Native Web: 0.17.7

**Comment by Warus15** [comment]
Hi, this issue should be fixed in [2.6.1](https://github.com/software-mansion/react-native-gesture-handler/releases/tag/2.6.1)

Note that this change is available in new web implementation. To enable it, call `enableExperimentalWebImplementation()` in the root of your project.

**Comment by j-piasecki** [comment]
I'll close the issue since the new web implementation is enabled by default starting with 2.10.0.
